regexes/client/browsers.yml

```diff
@@ -5,6 +5,130 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Mogok Browser (https://apps.apple.com/gb/app/mogok-browser-2/id1439011302)
+- regex: 'Mogok/(\d+\.[\.\d]+)'
+  name: 'Mogok Browser'
+  version: '$1'
+
+# TrueLocation Browser (https://apps.apple.com/us/app/truelocation-browser/id1463071203)
+- regex: '(?:IOS)?TrueLocationBrowser/(\d+\.[\.\d]+)'
+  name: 'TrueLocation Browser'
+  version: '$1'
+
+# Diigo Browser (https://apps.apple.com/ca/app/diigo-browser/id933773981)
+- regex: 'DiigoBrowser$'
+  name: 'Diigo Browser'
+  version: ''
+
+# OnBrowser Lite (on.browser)
+- regex: '.*OnBrowserLite(\d+\.[\.\d]+)'
+  name: 'OnBrowser Lite'
+  version: '$1'
+
+# Bluefy (https://apps.apple.com/us/app/bluefy-web-ble-browser/id1492822055)
+- regex: 'Bluefy/(\d+\.[\.\d]+)'
+  name: 'Bluefy'
+  version: '$1'
+
+# Vision Mobile Browser (https://en.wikipedia.org/wiki/Vision_Mobile_Browser)
+- regex: '(?:Novarra-Vision|Vision-Browser)(?:/(\d+[\.\d]+))?'
+  name: 'Vision Mobile Browser'
+  version: '$1'
+
+# Surfy Browser (https://outcoder.com/apps/surfy/)
+- regex: 'SurfyBrowser/(\d+[\.\d]+)'
+  name: 'Surfy Browser'
+  version: '$1'
+
+# 18+ Privacy Browser (https://play.google.com/store/apps/details?id=org.plus18.android)
+- regex: '18\+/([\d\.]+)'
+  name: '18+ Privacy Browser'
+  version: '$1'
+
+# GoKu (https://sites.google.com/view/goku-browser)
+- regex: 'GoKu-iOS/(\d+[\.\d]+)'
+  name: 'GoKu'
+  version: '$1'
+  engine:
+    default: 'WebKit'
+
+# Ask.com (https://play.google.com/store/apps/details?id=com.ask.browser)
+- regex: 'Ask\.com Mobile Browser'
+  name: 'Ask.com'
+  version: ''
+
+# Bang (com.tencent.bang)
+- regex: 'Bang/(\d+[\.\d]+)'
+  name: 'Bang'
+  version: '$1'
+
+# Intune Managed Browser (https://learn.microsoft.com/en-us/mem/intune/apps/manage-microsoft-edge)
+- regex: 'ManagedBrowser(?:/(\d+[\.\d]+))?'
+  name: 'Intune Managed Browser'
+  version: '$1'
+
+# Lotus (https://play.google.com/store/apps/details?id=dev.sect.lotus.browser.videoapp)
+- regex: 'Lotus/(\d+[\.\d]+)'
+  name: 'Lotus'
+  version: '$1'
+
+# JUZI Browser (https://www.123juzi.com/)
+- regex: 'JuziBrowser'
+  name: 'JUZI Browser'
+  version: ''
+
+# Ninetails (https://github.com/MystPi/ninetails)
+- regex: 'Ninetails(?:/(\d+[\.\d]+))?'
+  name: 'Ninetails'
+  version: '$1'
+
+# Wexond (https://github.com/wexond/browser-base)
+- regex: 'Wexond(?:/(\d+[\.\d]+))?'
+  name: 'Wexond'
+  version: '$1'
+
+# Catalyst (https://github.com/CatalystDevOrg/Catalyst)
+- regex: 'catalyst(?:/(\d+[\.\d]+))?'
+  name: 'Catalyst'
+  version: '$1'
+
+# Impervious Browser (https://www.impervious.ai/)
+- regex: 'Impervious(?:/(\d+[\.\d]+))?'
+  name: 'Impervious Browser'
+  version: '$1'
+  engine:
+    default: 'Gecko'
+
+# Moya Browser (https://play.google.com/store/apps/details?id=nu.bi.moya | https://apps.apple.com/za/app/moya-app-datafree/id1623208355)
+- regex: 'binu(?:/(\d+[\.\d]+))?'
+  name: 'Moya Browser'
+  version: ''
```

Comment:
According to the screenshots of the application, it does not look like a browser at all